internal/api/va/timeslot/remove/remove.go

```diff
@@ -0,0 +1,56 @@
+package remove
+
+import (
+	"context"
+	"vf-admin/internal/api"
+)
+
+// HTTPOperation abstracts away the current HTTP operation
+type HTTPOperation struct{}
+
+var vaID string
+var tsID string
+var authKey string
+
+// SetAuthKey sets the authentication key to be used for the HTTP operation
+func (HTTPOperation) SetAuthKey(key string) {
+	authKey = key
+}
+
+// SetRequestURLArguments sets the appropriate url arguments for the HTTP operation
+func (HTTPOperation) SetRequestURLArguments(args []string) error {
+	vaID = args[0]
+	tsID = args[1]
+
+	return nil
+}
+
+// GetDetails returns the details of the HTTP operation
+func (HTTPOperation) GetDetails() (string, string, string) {
+	return "remove", "removed", "va timeslot"
```

Comment:
you're missing attaching the id to the end for quite a few of them (put the tsID).